game_eggs/venice_unleashed/README.md

```diff
@@ -0,0 +1,53 @@
+﻿
+# [Venice Unleashed](https://veniceunleashed.net/)
+
+Experience Battlefield 3 like never before
+VU /vi:ju:/ is a community-oriented Battlefield 3 modding platform that gives you back control of your gaming experience.
+
+## PORTS
+
+ - 3 Ports are required to run Venice Unleashed servers
+ - Primary port is automatically assigned as game port for the server
+ - Harmony services and RCON needs port assigned in the Startup tab
+
+|  Port   |      Default Ports      |
+|---------|-------------------------|
+|  Game   |  Primary Port of server |
+| Harmony |           7948          |
+|  RCON   |          47200          |
+
+## EGG VARIANT
+
+There is 2 variants of Eggs for Venice Unleashed
+- With mount
+- Without mount
+
+### WITH MOUNT
+
+In order to reduce size of server, **/bf3** directory is shared with multiple servers rather than having it in root directory of each server.
+
+This type of egg uses mount feature of Pterodactyl where you need to create a mount for **/bf3** directory and mount it to server.
+Refer to this [link](https://pterodox.com/guides/mounts.html)  for more info on how to create mounts in Pterodactyl Panel
```

Comment:
Should use the official documentation for mounts: https://pterodactyl.io/guides/mounts.html

Pterodox is a community documentation site that has since been merged and abandoned (I'm one of the old maintainers).

Reply:
Oh.. Didn't know it was abandoned
Will update to use the official wiki